cmd/helm/flags.go

```diff
@@ -58,6 +58,10 @@ func addChartPathOptionsFlags(f *pflag.FlagSet, c *action.ChartPathOptions) {
 	f.StringVar(&c.CaFile, "ca-file", "", "verify certificates of HTTPS-enabled servers using this CA bundle")
 }
 
+func addStorageOptionsFlags(f *pflag.FlagSet, v *values.Options) {
+	f.StringArrayVarP(&v.Labels, "set-label", "l", []string{}, "specify labels for release secret/configmap on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)")
```

Comment:
since we want to take map[string]string , you can use something like...

```
f.StringToStringVar(&client.Labels, "labels", nil, "if set, adds these labels to release metadata")
```
as is done in #9087 , that would reduce some code introduced in this PR e.g.  `func (opts *Options) ParseLabels()` introduced in options.go and related unit tests.

Reply:
@liuming-dev did you get a chance to look at this?